package/utils/busybox/files/busybox-history-file.sh

```diff
@@ -0,0 +1,2 @@
+export HISTFILE=/tmp/.busybox_ash_history
```

Comment:
Does the resulting file need any special permissions treatment?

Reply:
Well, busybox creates the history file with tight permissions, owner only:

```
root@router6000:~# ls -l /tmp/.busybox_ash_history
-rw-------    1 root     root            67 Dec  5 23:17 /tmp/.busybox_ash_history
```

Reply:
OK. That would be my only concern then. Looks good.

Would it make sense to save it to flash upon reboot in `/root`? Although what you have here is 'good enough'.

Reply:
My proposal is intentionally ramdisk-only. 
Like I wrote in commit message, if somebody wants to save history over reboots, they can easily change the location without recompiling. /etc/profile.d/ is also saved over reboot, so any change you make there, gets carried over.